src/xproc3/schematron-xqs/catalog.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog" prefer="system">
+	
+	<system systemId="http://www.jenitennison.com/xslt/xspec/xproc/steps/run-schematron-xqs"
+		uri="xqs-harness.xproc" />
+	
+	<system systemId="http://www.jenitennison.com/xslt/xspec/xproc/steps/compile-schematron-xqs"
```

Comment:
Thanks for adding this extra catalog. :) My suggestion here is because this pipeline converts an XSpec test suite for Schematron into an XSpec test suite for XQuery, which is distinct from the (later) phase that converts an XSpec test suite into a test runner. I use the word preprocess for the first of those two phases and compile for the second.

```suggestion
	<system systemId="http://www.jenitennison.com/xslt/xspec/xproc/steps/preprocess-schematron-xqs"
```